### PR TITLE
Add dock static-only command

### LINKED ISSUE #39: Only show active applications in Dock
- [x] I made sure this command is not in the commands list
- [x] I searched and didn't find this command in the listed issues
- [x] I know this `defaults` command exists
- [x] I know this command works on macOS current version
- [ ] I know this command is deprecated on macOS current version

- Folder: Dock
- Command: defaults write com.apple.dock static-only -bool true && killall Dock
- Argument type (if applicable): bool
- Examples:
  - `true` enabled
  - `false` (default) disabled

### PR DESCRIPTION
**Pull request by thisisommore**
Fixes #39

### CODE REVIEW

**Comment by thisisommore** [comment]
@yannbertrand Do I need to have mac os to get screenshots and test this command?

**Comment by yannbertrand** [comment]
> @yannbertrand Do I need to have mac os to get screenshots and test this command?

@thisisommore Thanks for your PR, it's ok without it 🙂

**Comment by thisisommore** [comment]
@yannbertrand I have marked it as ready for review

File: defaults.yml [hunk 1]
```diff
@@ -194,6 +194,15 @@ categories:
...
         after: killall Dock
+      - key: static-only
+        domain: com.apple.dock
+        title: Only show active applications in Dock
+        param:
+          type: bool
+        examples:
+          - value: true
+            default: true
+        after: killall Dock
```
Comment (yannbertrand) [inline defaults.yml:205]: Add description and examples details

```suggestion
      - key: static-only
        domain: com.apple.dock
        title: Active applications only
        description: Only show opened apps in Dock
        param:
          type: bool
        examples:
          - value: true
             text: Only show active apps
          - value: false
             default: true
             text: Hide inactive apps from the Dock
        after: killall Dock
```

**Comment by yannbertrand** [comment]
LGTM, thanks @thisisommore :)

**Pull request merged**